### PR TITLE
Fix Transfer Details parsing

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#19780

*Description of changes:*
- Do not parse Transfer details when the frozen doc type is Transport Permit
- 
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1764-vfe7t6vc.web.app

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue [hunk 1]
```diff
@@ -893,8 +894,12 @@ export default defineComponent({
       setEmptyMhrTransfer(initMhrTransfer())
       setEmptyMhrTransportPermit(initTransportPermit())
 
+      const isFrozenDueToTransportPermit =
+        getMhrInformation.value.frozenDocumentType === UnitNoteDocTypes.TRANSPORT_PERMIT
+
       // Set baseline MHR Information to state
-      await parseMhrInformation(isFrozenMhr.value)
+      // Do not parse Transfer details when frozen doc type is Transport Permit - bug 19780
```
Comment (cameron-eyds) [inline ppr-ui/src/views/mhrInformation/MhrInformation.vue]: You want to eternalize the ticket number in the code comment?

File: ppr-ui/src/utils/feature-flags.ts [hunk 2]
```diff
@@ -10,12 +10,12 @@ export const defaultFlagSet: LDFlagSet = {
...
-  'mhr-transfer-enabled': false, // Enables changes to base MHR HomeOwners within the MHR Information flow
-  'mhr-exemption-enabled': false,
-  'mhr-transport-permit-enabled': false,
-  'mhr-amend-transport-permit-enabled': false,
+  'mhr-registration-enabled': true, // Enables MHR table tab
+  'mhr-staff-correction-enabled': true, // Enables access to staff mhr correction
+  'mhr-transfer-enabled': true, // Enables changes to base MHR HomeOwners within the MHR Information flow
+  'mhr-exemption-enabled': true,
+  'mhr-transport-permit-enabled': true,
+  'mhr-amend-transport-permit-enabled': true,
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/feature-flags.ts]: Defaults

**Review by cameron-eyds — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: To reference the issue, for tricky bugs it is helpful. We can look up the commit and track it that way, but sometimes it gets overwritten by upgrades or other tickets.

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: Well it sets a new precedent, were going to do this for all bugs? Tricky bugs? Whats a tricky bug? 

Either way, i disagree with this when your editor can give you this information with a click but whatever.

File: ppr-ui/src/utils/feature-flags.ts (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/utils/feature-flags.ts]: Oops, thanks, removed.

File: ppr-ui/src/views/mhrInformation/MhrInformation.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: A tricky bug is when it takes 3 people to figure out the issue, and come up with a workaround like this when there is no straightforward fix. It's ok to reference bugs. And as I said, sometimes it gets overwritten with other commits.

Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrInformation.vue]: What gets overwritten? Git History tied to commits, tied to PRs? I don't think so

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1764-vfe7t6vc.web.app

**Pull request merged**